apps/web/screens/ServiceDetails/ServiceDetails.tsx

```diff
@@ -154,8 +145,8 @@ const ServiceDetails: Screen<ServiceDetailsProps> = ({
                       { title: n('linkIslandIsText'), href: n('linkIslandIs') },
                       { title: n('linkThrounText'), href: n('linkThroun') },
                       {
-                        title: n('linkServiceListText'),
-                        href: n('linkServiceList'),
+                        title: n('linkServicesText'),
+                        href: n('linkServices'),
```

Comment:
Using linkResolver for this link might remove the need for this namespace variable:
```ts
const { linkResolver } = useLinkResolver() // at the top of this cmp
...
{
  title: title: n('linkServicesText'),
  href: linkResolver('webservicespage').as
```
Also passing href as an object here might allow the cmp using this data to have prefetch by using the returned `as` and `href` properties